packages/ui/menu/index.ts

```diff
@@ -0,0 +1 @@
+export * from './menu';
```

Comment:
I don't think we need this file, I don't see anyone importing from `@ifixit/ui/menu`

Reply:
I'll test it without this file, but I believe we do need it. `/packages/ui/index.tsx` has `export * from './menu';` which points to `packages/ui/menu/index.ts`. So I think it is a necessary intermediary export.